Unreal Launcher/Unreal Launcher/MainWindow.xaml.cs

```diff
@@ -40,7 +40,7 @@ public MainWindow()
 
 			if (Settings.Default.Projects != null)
 			{
-				System.Collections.Specialized.StringCollection InvalidProjects = new System.Collections.Specialized.StringCollection();
+				System.Collections.Specialized.StringCollection invalidProjects = new System.Collections.Specialized.StringCollection();
 
 				foreach (string projectstr in Settings.Default.Projects)
```

Comment:
Another variable which might be nicer as, say, `projectName` or `projectKey` or .. just something more descriptive than `projectstr`